src/index.html

```diff
@@ -3,11 +3,311 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="shortcut icon"
+      href="images/favicon.ico"
+      type="image/x-icon"
+    >
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <div class="header__wrapper">
+        <div class="header__menu">
```

Comment:
once covered on lesson, please implement
>17	Fix menu for small screens (if there is not enough space for all the menu items)
>18	Disable page scrolling under the menu

Reply:
17: Menu fits in all resolutions, even down to 320 x 640. Point 17 thus not implemented.

18: Done! What I have noticed though, is that the pointer shifts to the right by the scrollbar with (18px I believe). I'm wondering is there a way to compensate for this other then changing right: 18px?